runtime/perfetto/db.go

```diff
@@ -142,6 +156,17 @@ CREATE TABLE IF NOT EXISTS replica_num (
 	PRIMARY KEY(app,version,weavelet_id)
 );
 `
+	if opts.MaxTraceBytes > 0 {
+		initTables += `
+-- Single-row table that stores the total size of all trace data.
```

Comment:
We should try to get rid of this size maintenance. Perhaps by looking at the size of the perfetto.db file?

Apart from code complexity, this is also imposes an invariant across tables that could be violated by somebody coming along and deleting or inserting trace data directly.

Reply:
Done, this is much cleaner now. PTAL.